src/main/kotlin/racingcar/strategy/UnconditionalMovingStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.strategy
+
+class UnconditionalMovingStrategy: MovingStrategy {
+    override fun canMove(): Boolean = true
+}
```

Comment:
Test를 위한 Fake객체라면 Test 패키지에서 생성해도 좋을거같아요! 
```suggestion
class FakeMovingStrategy(val canMove: Boolean) : MovingStrategy {
    override fun canMove(): Boolean = canMove
}
```
위같은 형태로 전진만하는 FakeStrategy, 전진안하는 Strategy 모두 생성할수 있을거같아요!